sdk/containerregistry/azure-containerregistry/samples/async_samples/sample_hello_world_async.py

```diff
@@ -48,7 +48,7 @@ async def basic_sample(self):
                         print(tag.digest)
 
                     # [START delete_repository]
-                    await client.delete_repository(repository_name, "hello-world")
```

Comment:
Was this sample previously broken?

Reply:
delete_repository() only accept one positional value but we provide two here, it should broken.
But, YES- all samples, not only this one, are broken as lack of resources, they all need some images uploaded in advance. I have updated this step in the samples PR.